### ISSUE TITLE
Issue with running the code

### ISSUE THREAD

**Issue by rishika2110** [lsl001006/ZONE#4]
Hi, I am facing a few issues while running your code. I am getting the following error when trying to run inferece.py.

```
`text_config_dict` is provided which will be used to initialize `CLIPTextConfig`. The value `text_config["id2label"]` will be overriden.
unet/diffusion_pytorch_model.safetensors not found
Traceback (most recent call last):
  File "inference.py", line 155, in <module>
    fused_ip2p(args)
  File "inference.py", line 36, in fused_ip2p
    sam_predictor, _, pipe, mb_pipe = model_init_insert(args)
  File "inference.py", line 22, in model_init_insert
    mb_pipe = StableDiffusionInstructPix2PixPipeline.from_pretrained(
  File "/home/rishika.bhagwatkar/miniconda3/envs/zone/lib/python3.8/site-packages/diffusers/pipelines/pipeline_utils.py", line 1039, in from_pretrained
    loaded_sub_model = load_sub_model(
  File "/home/rishika.bhagwatkar/miniconda3/envs/zone/lib/python3.8/site-packages/diffusers/pipelines/pipeline_utils.py", line 445, in load_sub_model
    loaded_sub_model = load_method(os.path.join(cached_folder, name), **loading_kwargs)
  File "/home/rishika.bhagwatkar/miniconda3/envs/zone/lib/python3.8/site-packages/diffusers/models/modeling_utils.py", line 583, in from_pretrained
    raise ValueError(
ValueError: Cannot load <class 'diffusers.models.autoencoder_kl.AutoencoderKL'> from /home/rishika.bhagwatkar/scratch/.cache/huggingface/hub/models--vinesmsuic--magicbrush-jul7/snapshots/d75a9f18439f67dbdc0ab113036f491ce2256c53/vae because the following keys are missing: 
 encoder.mid_block.attentions.0.value.bias, decoder.mid_block.attentions.0.proj_attn.weight, decoder.mid_block.attentions.0.query.bias, decoder.mid_block.attentions.0.key.weight, decoder.mid_block.attentions.0.value.weight, decoder.mid_block.attentions.0.value.bias, encoder.mid_block.attentions.0.query.weight, decoder.mid_block.attentions.0.proj_attn.bias, encoder.mid_block.attentions.0.proj_attn.bias, encoder.mid_block.attentions.0.proj_attn.weight, encoder.mid_block.attentions.0.query.bias, decoder.mid_block.attentions.0.query.weight, decoder.mid_block.attentions.0.key.bias, encoder.mid_block.attentions.0.value.weight, encoder.mid_block.attentions.0.key.weight, encoder.mid_block.attentions.0.key.bias. 
 Please make sure to pass `low_cpu_mem_usage=False` and `device_map=None` if you want to randomly initialize those weights or else make sure your checkpoint file is correct.
```

The command I used was `python inference.py --instruction "remove the collar of the dog" --image_path "WaterproofCollar_Black_Front.jpg"`

I followed everything as mentioned in the readme. However, I had to loosen up some package versions as I was getting a conflict of numpy version across packages.

I would appreciate any help from your side :)

**Comment by lsl001006** [comment]
Hi,

It appears that the issue is with the missing file: “unet/diffusion_pytorch_model.safetensors.” It looks like you’re importing the autoencoder from ‘magicbrush’ instead of ‘instructpix2pix.’ Please ensure that the checkpoints for both ‘instructpix2pix’ and ‘magicbrush’ are correctly installed. If the problem persists, feel free to comment on this issue.

Best regards,

**Comment by Amaia-CARDIEL** [comment]
Hi,

Thank you for your great work !

I'm commenting this issue as I ran into the same error while trying to run your code. 

I ended up finding that when installing requirements, there was a conflict between lama-cleaner==1.2.3 and diffusers==0.18.0. As a result, diffusers==0.16.1 has been installed instead and this created the above error as some implementation changed between diffusers 0.16 and 0.18. I re-installed diffusers==0.18.0, ignored the conflict with lama-cleaner==1.2.3 and now inference code works.

Hope this may help other people. 

Best regards
